pages/vi/profiles/seantang001.md

```diff
@@ -0,0 +1,15 @@
+# Sean Tang
+___
+## Basic Info
+**Name**: Sean Tang  
+**Location**: CA, USA  
+**Time Zone**: Pacific Time  
+**OS**: Ubuntu 18.04 (bionic)  
+**Description:**  
+My name is Sean Tang. I'm currently a Junior attending a high school in the Bay Area. I aspire to become a computer scientist in the future and I look forward to working with you guys! I've worked with android apps and raspberry pi in the past. Please feel free to check out my github profile!  
```

Comment:
@SeanTang001 this is a very good start
maybe as 5 markdown element you add the link to your [github profile]()

Reply:
Yes. Thanks for the advice!